src/icons/svg/ActionsArrowDown.svg

```diff
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: sketchtool 63.1 (101010) - https://sketch.com -->
     <title>icon/actions_arrow_down</title>
-    <desc>Created with sketchtool.</desc>
```

Comment:
@akdetrick This got removed from every file. I'm almost 100% sure this is related to having the newest version of Sketch, so we might both want to be running the newest Sketch app.